web/src/components/ui/card.tsx

```diff
@@ -1,6 +1,6 @@
-import * as React from "react";
+import * as React from "react"
```

Comment:
These changes where made while installing the new chart lib. They seem safe enough to not affect other things within the code.